lib/code_corps/sparkpost/emails/forgot_password.ex

```diff
@@ -0,0 +1,26 @@
+defmodule CodeCorps.SparkPost.Emails.ForgotPassword do
+  alias CodeCorps.{User, WebClient}
+  alias SparkPost.{Content, Transmission}
+
+  @spec build(User.t, String.t) :: %Transmission{}
+  def build(%User{} = user, token) do
+    %Transmission{
+      content: %Content.TemplateRef{template_id: "forgot-password"},
+      options: %Transmission.Options{inline_css: true},
+      recipients: [%{name: user.first_name, email: user.email}],
+      substitution_data: %{
+        from_name: "Code Corps",
+        from_email: "team@codecorps.org",
```

Comment:
These are identical everywhere, as far as I can tell, so maybe we add a helper module which builds the base model.